src/cascadia/WindowsTerminal/NonClientIslandWindow.cpp

```diff
@@ -590,7 +590,7 @@ void NonClientIslandWindow::_UpdateFrameMargins() const noexcept
         return 0;
     }
 
-    PAINTSTRUCT ps{ 0 };
+    PAINTSTRUCT ps{ nullptr };

```

Comment:
This is maybe one where I _wouldn't_ use `nullptr`, IMO it makes more sense to be "zero initializing" this struct rather than "`nullptr` initializing", but that's absolutely a nit that doesn't _practically_ matter